packages/vats/test/test-lib-chainStorage.js

```diff
@@ -68,7 +75,7 @@ test('makeChainStorageRoot', async t => {
     );
   }
 
-  rootNode.clearValue();
+  rootNode.setValue(undefined);
```

Comment:
This call is invalid.
```suggestion
  rootNode.setValue('');
```